moninag/static/src/ts/app/servers/edit-server.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { ActivatedRoute, Params, Router } from '@angular/router';
+import { Location } from '@angular/common';
+import 'rxjs/add/operator/switchMap';
+
+
+import { Server, states } from './model';
+import { ServersService } from './service';
+import { ServicesComponent } from '../services/services.component';
+
+
+@Component({
+    selector: 'servers-edit',
+    template: `
+    <div *ngIf="server"><div>
+    <h5>Server:</h5>
+    <input [(ngModel)]="server.name" placeholder="{{server.name}}" />
+    <h5>Address:</h5>
+    <input [(ngModel)]="server.address" placeholder="{{server.address}}" />
+    <h5>State:</h5>
```

Comment:
use label instead of h5 and wrap it together with select field i a `.form-group`